src/bin/simhash-find-all.cpp

```diff
@@ -0,0 +1,204 @@
+#include <iostream>
+#include <unordered_set>
+#include <sstream>
+#include <fstream>
+
+#include <getopt.h>
+
+#include "simhash.h"
+
+void usage(int argc, char** argv)
+{
+    std::cout << "usage: " << argv[0]
+              << " --blocks BLOCKS"
+              << " --distance DISTANCE"
+              << " --input INPUT"
+              << " --output OUTPUT"
+              << std::endl
```

Comment:
I think either is fine from an idiomatic perspective. In this particular case, I think `\n` will make it a little less cluttered and readable.